### ISSUE TITLE
Feature: Official vs community PHP support

### ISSUE THREAD

**Issue by alexellis** [openfaas/faas#783]
OpenFaaS can support any language due to our focus on portability, but with each "supported" language there is a cost to the regular contributors. I want to gauge the interest levels in moving a PHP template into the [official repository](https://github.com/openfaas/templates).

## Existing community-supported PHP templates

@itscaro put together the "original" PHP template here:

https://github.com/itscaro/openfaas-template-php/

@andrew-s has put together a newer option for PHP 7.x (I have not tested this)

https://github.com/itscaro/openfaas-template-php/compare/master...seymourlabs:master

**Comment by itscaro** [comment]
Cool 👍 I will have a look @alexellis @andrew-s

**Comment by Brunty** [comment]
I'd use this for personal stuff, work is a possibility too

**Comment by alexellis** [comment]
@andrew-s let's look at a PR to the templates repo to add your template there for the latest PHP version?

**Comment by andrew-s** [comment]
@alexellis Yeah, sounds good - let me know what you need and I'll get it going.

**Comment by Brunty** [comment]
If there’s anything I can help with let me know :)

**Comment by alexellis** [comment]
How about a PR to the templates repo? Please just checkout the contribution guide in the faas repo before starting that though.

**Comment by andrew-s** [comment]
Apologies for the delay, see the PR at; https://github.com/openfaas/templates/pull/65

**Comment by alexellis** [comment]
Thanks Andrew, I see some people have given feedback given on the PR. I've also left some messages.

**Comment by alexellis** [comment]
@Brunty @ssx thanks for volunteering to help. Please can you both go ahead and take the template for a test-drive?

**Comment by ssx** [comment]
Haven't had a chance to look at this yet as I've been away from a computer for a short while. 

Will take a look this week and provide feedback.
